.build/run-tests.sh

```diff
@@ -240,6 +241,9 @@ _main() {
       fi
       ant testclasslist -Dtest.classlistprefix=distributed -Dtest.timeout=$(_timeout_for "test.distributed.timeout") -Dtest.classlistfile=<(echo "${testlist}") ${ANT_TEST_OPTS} || echo "failed ${target} ${split_chunk}"
       ;;
+    "build_dtest_jars")
```

Comment:
Should we add `dtest-latest` to the `run-tests.sh` as well?

Reply:
It is:
- https://github.com/apache/cassandra/blob/19509dbd623af9fdbf1696694e41034b9b6b40f6/.build/run-python-dtests.sh#L120-L121
- https://github.com/apache/cassandra/blob/19509dbd623af9fdbf1696694e41034b9b6b40f6/.build/docker/run-tests.sh#L147

Reply:
I meant this one `.build/run-tests.sh`, but now I see that it's not designed for the dtests.